subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/IntentionExecutionServiceImplementation.kt

```diff
@@ -65,16 +65,17 @@ class IntentionExecutionServiceImplementation(
                         timeProvider = timeProvider
                     ).run(scope)
                 }
+                throw IllegalStateException("devices channel was closed")
```

Comment:
I've decided that IlligalStateException, when `devices` is closed, is better than closing the whole scope